contrib/stable/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# stable/Dockerfile
+#
+# Build a Skopeo container image from the latest
+# stable version of Skopeo on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=skopeo
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
+
+# Don't include container-selinux and remove
+# directories used by yum that are just taking
+# up space.
+RUN useradd build; yum -y update; yum -y reinstall shadow-utils; yum -y install skopeo fuse-overlayfs --exclude container-selinux; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
```

Comment:
What does the `reinstall` do?

Reply:
For some reason fedora base images loose the setuid/setgid bits on newuidmap and newgidmap.  This makes sure the files have the proper file caps

Reply:
That would be useful to document in the file itself.

Reply:
added doc, good thought.